src/resources/kyc-documents-resource.js

```diff
@@ -1,46 +1,34 @@
-export default function KycDocumentsResource({apiHandler}) {
-    return {
-        getAll({limit = null, offset = null, sort = null, expand = null, filter = null, q = null} = {}) {
-            const params = {
-                limit,
-                offset,
-                sort,
-                expand,
-                filter,
-                q
-            };
-            return apiHandler.getAll(`kyc-documents`, params);
-        },
-
-        get({id, expand = null}) {
-            const params = {expand};
-            return apiHandler.get(`kyc-documents/${id}`, params);
-        },
-
-        create({data, expand = null}) {
-            const params = {expand};
-            return apiHandler.create(`kyc-documents`, '', data, params);
-        },
-
-        update({id, data, expand = null}) {
-            const params = {expand};
-            return apiHandler.put(`kyc-documents/${id}`, data, params);
-        },
+/**
+* This file was auto-generated by rebilly-sdk-generator.
+* Do not make direct changes to this file.
+*/
 
-        accept({id}) {
-            return apiHandler.post(`kyc-documents/${id}/acceptance`);
-        },
-
-        reject({id, data}) {
-            return apiHandler.post(`kyc-documents/${id}/rejection`, data);
-        },
-
-        review({id}) {
-            return apiHandler.post(`kyc-documents/${id}/review`);
-        },
-
-        matches({id, data}) {
-            return apiHandler.post(`kyc-documents/${id}/matches`, data);
-        },
-    };
-};
+export default function KycDocumentsResource({apiHandler}) {
+  return {
+    getAll({limit = null, offset = null, filter = null, sort = null} = {}) {
+      const params = {limit, offset, filter, sort};
+      return apiHandler.getAll(`kyc-documents`, params);
+    },
+    create({id = '', data}) {
+      return apiHandler.create(`kyc-documents/${id}`, id, data);
+    },
+    get({id}) {
+      return apiHandler.get(`kyc-documents/${id}`);
+    },
+    update({id, data}) {
+      return apiHandler.put(`kyc-documents/${id}`, data);
+    },
```

Comment:
`expand` is missing in the params of these three methods, is this ok?

Reply:
We don't need `expand`. This PR fixes that discrepancy.